Sources/metalgpu/MetalGpuInfo.swift

```diff
@@ -216,6 +237,8 @@ extension MTLDevice {
                 .macCatalyst1: "MacCatalyst1",
                 .macCatalyst2: "MacCatalyst2"
             ]
+            
+            if #available(macOS 13.0, *) {families[.metal3] = "Metal3"}
```

Comment:
I prefer this style :)

```suggestion
            if #available(macOS 13.0, *) {
                families[.metal3] = "Metal3"
            }
```